index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Microverse/freeCodeCamp Survey Form</title>
+</head>
+
+<body>
+  
+  <h1 id="title">freeCodeCamp Survey Form</h1>
+
+  <P id="description">Thank you for taking the time to help us improve the platform</P>
```

Comment:
Kindly wrap your h1 and p elements and the form element as well with a section tag because of semantics reasons. and SEO.

Reply:
Great job @MostafaShahat2021 🤝

- [ ] [Optional] Kindly use emojis as they make the communication human-friendly.

Reply:
Hi @1GORDON, thanks a lot for your efforts in reviewing my code, now should I submit the PR link or merge changes to the main branch and submit the repo link?
Thanks in advance,
Best Regards